### PR TITLE
Dataset 3 Phase 1 Task 72-95

### PR DESCRIPTION
**Pull request by AnshitaSinghBais**

### CODE REVIEW

**Comment by Ravsehajsinghpuri** [comment]
@AnshitaSinghBais Can you rename the files as per the correct format I mentioned in the other PR?

File: tasks/README.md [hunk 1]
```diff
@@ -1,9 +1,75 @@
-## Tasks List 
-
-This directory contains the tasks that are part of this benchmark. 
-
-
-Name | Summary | Category
----- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation
```
Comment (Ravsehajsinghpuri) [inline tasks/README.md:8]: add this task

File: tasks/README.md [hunk 2]
```diff
@@ -1,9 +1,75 @@
-## Tasks List 
-
-This directory contains the tasks that are part of this benchmark. 
-
-
-Name | Summary | Category
----- | ----------- | --------
-`task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
```
Comment (Ravsehajsinghpuri) [inline tasks/README.md:9]: add this task

**Comment by Ravsehajsinghpuri** [comment]
@AnshitaSinghBais pls resolve the conflict, do a git pull to get the updated 'main' and then commit your changes to remove the conflicts

**Comment by AnshitaSinghBais** [comment]
> @AnshitaSinghBais pls resolve the conflict, do a git pull to get the updated 'main' and then commit your changes to remove the conflicts

The HW3 branch was 638 commits behind the main branch and wasn't able to pull the 'main' branch. I am trying to create a new pull request with a new branch for phase 1. Do I have to do this for phase 2 also? As you reversed my phase 2 PR.

**Comment by AnshitaSinghBais** [comment]
> @AnshitaSinghBais pls resolve the conflict, do a git pull to get the updated 'main' and then commit your changes to remove the conflicts

@Ravsehajsinghpuri 
I have created a new PR for both phase 1 and phase 2 with no conflicts. Please consider those. Sorry for the issues caused.

Thank you

**Comment by Ravsehajsinghpuri** [comment]
@AnshitaSinghBais you can close this PR, since you have opened a new one for the same

**Pull request closed without merging**